ERCS/erc-9999.md

```diff
@@ -0,0 +1,186 @@
+---
+eip: 9999
```

Comment:
```suggestion
eip: 7644
```
Assigning next sequential EIP/ERC/RIP number.
Numbering changed to sequential from 7500.

Please also update the filename.